File: java/Sers.ServiceStation/src/Sers/Core/Module/Api/ApiDesc/SsModelProperty.java
package Sers.Core.Module.Api.ApiDesc;

public class SsModelProperty {

    /**
         * 名称
     */
    public String name;

    /**
         * 数据类型。 可为 string、int32、int64、float、double、bool、datetime 或 SsModelEntity的name
     */
    public String type;

    /**
         * 数据模式。只可为 value、object、array
     */
    public String mode;

    /**
         * 描述
     */
    public String description;

    /**
         * 默认值
     */
    public Object defaultValue;

    /**
         * 示例值
     */
    public Object example;
}

File: Sers/Sers/Sers.Core/Sers.Core.Station/Module/Api/ApiDesc/SsModelProperty.cs
using Newtonsoft.Json;

namespace Sers.Core.Module.Api.ApiDesc
{
    [JsonObject(MemberSerialization.OptIn)]
    public class SsModelProperty
    {
        /// <summary>
        /// 名称
        /// </summary>
        [JsonProperty]
        public string name;

        /// <summary>
        /// 数据类型。 可为 string、int32、int64、float、double、bool、datetime 或 SsModelEntity的name
        /// </summary>
        [JsonProperty]
        public string type;

        /// <summary>
        ///  数据模式。只可为 value、object、array
        /// </summary>
        [JsonProperty]
        public string mode;

        /// <summary>
        /// 描述
        /// </summary>
        [JsonProperty]
        public string description;

        /// <summary>
        /// 默认值
        /// </summary>
        [JsonProperty]
        public object defaultValue;

        /// <summary>
        /// 示例值
        /// </summary>
        [JsonProperty]
        public object example;
    }
}
